### PR TITLE
[FIX] SRP recover when an error is thrown

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
The SRP was not possible to recover when an error was found.

**Proposed solution**
SRP is able to be recovered when an error is found

**How to test it**
* Setup dev environment
* Write this code line `throw new Error('testing')` on the beginning of [wallet/index.tsx](https://github.com/MetaMask/metamask-mobile/blob/e05ef5ee785f171ba994373ebde4128e2dd18d20/app/components/Views/Wallet/index.tsx#L72) 
* Reload the app
* Insert password
* Expected to see the Error boundary screen
* Expected to be able to recover password
* (ONLY ON ANDROID)
* Try to take a screenshot of SRP
* Expected behaviour is that it's not possible

(This screen doesn't have the warning alert of when the screenshot is taken on IOS)

**Screenshots/Recordings**
https://recordit.co/dbQHB41jPb

**Issue**

Progresses #

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by gantunesr — CHANGES_REQUESTED** [review]
This is a strange solution to this problem, lets sync on it later

File: app/components/Views/Root/index.js [hunk 1]
```diff
@@ -26,10 +26,6 @@ export default class Root extends PureComponent {
     foxCode: 'null',
   };
 
-  errorHandler = (error, stackTrace) => {
```
Comment (sethkfman) [inline app/components/Views/Root/index.js:29]: Why did you remove this log?

Comment (tommasini) [reply app/components/Views/Root/index.js:29]: ErrorBoundary doesn't have any prop called onError, it was unused code

File: app/components/Views/ErrorBoundary/index.js [hunk 2]
```diff
@@ -232,9 +232,9 @@ class ErrorBoundary extends Component {
   render() {
     return this.state.backupSeedphrase ? (
       <RevealPrivateCredential
-        navBarDisabled
-        privateCredentialName={'seed_phrase'}
+        credentialName={'seed_phrase'}
         cancel={this.cancelExportSeedphrase}
+        errorBoundaryView={this.props.view}
```
Comment (gantunesr) [inline app/components/Views/ErrorBoundary/index.js:237]:
```suggestion
        hasNavigation={this.props.view === ROOT}
```

**Review by gantunesr — APPROVED** [review]
LGTM, just a minor comment

File: app/components/Views/ErrorBoundary/index.js (hunk 2 above)
Comment (tommasini) [reply app/components/Views/ErrorBoundary/index.js]: Done 🚀

**Review by gantunesr — APPROVED** [review]

**Review by cortisiko — APPROVED** [review]
this was QA passed 🌮 🌮

**Pull request merged**